Makefile

```diff
@@ -474,62 +474,35 @@ LOCALBIN ?= $(BUILD_AUX)/bin
 $(LOCALBIN):
 	mkdir -p $(LOCALBIN)
 
+# External generation tool versions
+CLIENT_GEN_VERSION := v0.31.2
+CONTROLLER_GEN_VERSION := v0.16.5
+
 ## External generation tool binaries
 CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
 CLIENT_GEN ?= $(LOCALBIN)/client-gen
 
-GIT_CLONE    := git clone
+# client generation rules
+.PHONY: controller-gen
+controller-gen: ## install controller-gen
```

Comment:
Nit: the ##-suffix does not make sense for us, because we lack the corresponding `make help` rule which would grep those out from the Makefile and produce a nice help page. Let's take that out, let's add the help-rule and then annotate the top-level targets with the help messages, all in part of a single PR.